### PR TITLE
Ks 2023 01 several tokens in session

### PR DESCRIPTION
**Pull request by Rineee**

### CODE REVIEW

File: meinberlin/apps/budgeting/serializers.py [hunk 1]
```diff
@@ -85,23 +85,27 @@ def get_session_token_voted(self, proposal):
...
+                            module=module,
+                        )
+                    except VotingToken.DoesNotExist:
+                        pass
 
                 vote = TokenVote.objects.filter(
                     token=token,
                     content_type=ContentType.objects.get_for_model(proposal.__class__),
                     object_pk=proposal.pk,
                 )
```
Comment (fuzzylogic2000) [inline meinberlin/apps/budgeting/serializers.py:107]: This block can be indented one level more, right? If the module is not in the voting_tokens, then we will also not find a vote anyway.

File: meinberlin/apps/votes/filters.py [hunk 2]
```diff
@@ -9,25 +9,23 @@ class OwnVotesFilterBackend(BaseFilterBackend):
...
 
                     if token:
-                        own_votes = TokenVote.objects.filter(
-                            token=token
-                        )
-                        return queryset.filter(
-                            id__in=own_votes.values("object_pk")
-                        )
+                        own_votes = TokenVote.objects.filter(token=token)
+                        return queryset.filter(id__in=own_votes.values("object_pk"))
```
Comment (fuzzylogic2000) [inline meinberlin/apps/votes/filters.py:29]: This if can also be indented one more

**Review by fuzzylogic2000 — APPROVED** [review]
Nice! Just two nit-picks

File: meinberlin/apps/budgeting/serializers.py (hunk 1 above)
Comment (Rineee) [reply meinberlin/apps/budgeting/serializers.py:107]: Ah yes, true!

**Review by fuzzylogic2000 — APPROVED** [review]
Nice!

**Pull request merged**